dnsapi/dns_ispconfig.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env sh
+
+#ISPConfig 3.1 API - Add remote user and give him access to at least the "DNS txt functions"
+
+# User must provide login data and URL to the ISPConfig installation incl. port. The remote user in ISPConfig must have access to the dns_txt_function
+# Values to export:
+
+# export ISPC_User="remoteUser"
+# export ISPC_Password="remotePasword"
+# export ISPC_Api="https://ispc.domain.tld:8080/remote/json.php"
+
+########  Public functions #####################
+
+#Usage: dns_myapi_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_ispconfig_add() {
+  fulldomain="${1}"
+  txtvalue="${2}"
+  _ISPC_login && _ISPC_getZoneInfo && _ISPC_addTxt || return 1
```

Comment:
see here: https://github.com/Neilpang/acme.sh/blob/master/dnsapi/dns_cf.sh#L17

we must save the CF_Key and CF_Email.  When acme.sh renew the cert automatically,  they will be needed.

So, you must save  ISPC_User  and   ISPC_Password